src/ui/public/courier/__tests__/request_queue.js

```diff
@@ -2,21 +2,20 @@ import ngMock from 'ng_mock';
 import expect from 'expect.js';
 import sinon from 'sinon';
 
-import { RequestQueueProvider } from '../_request_queue';
+import { requestQueue } from '../_request_queue';
 import { SearchStrategyProvider } from '../fetch/strategy/search';
 import { DocDataStrategyProvider } from '../fetch/strategy/doc_data';
 
 describe('Courier Request Queue', function () {
   let docStrategy;
-  let requestQueue;
   let searchStrategy;
 
   beforeEach(ngMock.module('kibana'));
   beforeEach(ngMock.inject(function (Private) {
     docStrategy = Private(DocDataStrategyProvider);
-    requestQueue = Private(RequestQueueProvider);
     searchStrategy = Private(SearchStrategyProvider);
   }));
+  beforeEach(requestQueue.clear);
```

Comment:
__nit:__ Would `afterEach` be more suitable here? At least logically, technically it doesn't matter unless there is a test somewhere that doesn't cleanup after itself properly. And the same note for the rest of the tests in this PR.

Reply:
What if I add both?  Then it's very safe, otherwise it might be hard to find all the test places that modify requestQueue indirectly.

Reply:
Though that might be overkill.  Maybe a beforeEach and an after, to run at the very end.